config.yaml

```diff
@@ -4,13 +4,12 @@
 # If your charm does not require configuration options, delete this file entirely.
 #
 # See https://juju.is/docs/config for guidance.
-
 options:
-  # An example config option to customise the log level of the workload
-  log-level:
-    description: |
-      Configures the log level of gunicorn. 
-
-      Acceptable values are: "info", "debug", "warning", "error" and "critical"
-    default: "info"
+  tls-secret-name:
+    default: ""
+    description: The name of the Kubernetes secret used to get the TLS certificate used for HTTPS. This settings overrides any tls-certificate relation defined. If left empty and no tls-certificates relation is defined, the charm will block until TLS is configured.
+    type: string
+  service-hostname:
```

Comment:
Can we alphabetise the config options, so this would come before tls-secret-name. Also, I realise we're using the same approach as nginx-ingress-integrator here, but I think we should possible consider a different name here. Either `external-hostname` or `hostname`. This will probably need some discussion with a few folks before we can confirm what we want, which might be easiest outside of this PR.

Reply:
as discussed internally, I updated this to `external-hostname`